CMakeLists.txt

```diff
@@ -295,8 +295,8 @@ list(APPEND FORT90_LIB
   bouncy_castle plato_seq bdex fma dabnew end_sixtrack
   ranecu wire elens mod_dist scatter dump dynk cheby utils
   aperture mod_fluc beam6d_fox six_fox postprocessing
-  read_write ranlux collimation coll_db coll_dist mod_particles
-  mod_ffield mod_ffield_aux
+  read_write ranlux collimation coll_db mod_particles cheby utils
+  mod_ffield mod_ffield_aux mod_find coll_dist
```

Comment:
`cheby` and `utils` were already in the list and are now doubled with your addition

Reply:
It's not an "addition" as much as it was a merge conflict. There is no harm in this, but I'll clean it up anyway.